Makefile

```diff
@@ -98,19 +98,33 @@ docker-pause:
 	DOCKER_BUILDKIT=1 docker build . -t malvag/pause:1.1.9 -f deploy/images/pause-apptainer-agent/pause.apptainer.Dockerfile
 	sudo docker push malvag/pause:1.1.9
 
+docker-kubemaster-k3s:
```

Comment:
- Just name this `image-kubemaster` (or even rename all instances of "kubemaster" to "hpk-master" so we use one naming scheme).
- Use a variable to upload to CARV's image repository (see other projects).
- Create one more rule to build all images.